Editor/Helper/AddressablesImporterMethodHandler.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace UnityAddressableImporter.Helper.Internal
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using System.Text.RegularExpressions;
+    using UnityEditor;
+    using UnityEngine;
+
+    public static class AddressablesImporterMethodHandler
```

Comment:
Please rename this class and this file to `AddressableImporterMethodHandler` (without the `s` to keep consistent with all file naming in this project).

Reply:
Not fix yet.